app/components/more-details.jsx

```diff
@@ -0,0 +1,139 @@
+// https://github.com/EnCiv/civil-pursuit/issues/89
+
+'use strict'
+import React, { useState } from 'react'
+import Column from './util/column'
+import Submit from './util/submit'
+import cx from 'classnames'
+import { createUseStyles } from 'react-jss'
+import { JsonForms } from '@jsonforms/react'
+import { vanillaCells, vanillaRenderers } from '@jsonforms/vanilla-renderers'
+
+const MoreDetails = props => {
+  const {
+    point = { subject: '', description: '', _id: '' },
+    defaultValue = { subject: '', description: '' },
+    onDone = () => {},
+    className,
+    ...otherProps
+  } = props
+  const [data, setData] = useState(initialData)
+  const classes = useStyles()
+  const initialData = {}
+
+  const schema = {
+    type: 'object',
+    properties: {
+      householdIncome: {
+        title: 'Household Income',
+        type: 'string',
+        enum: ['0-10000', '10000-20000', '20000-30000'],
+        default: 'Select household range',
+      },
+      housing: {
+        title: 'Housing',
+        type: 'string',
+        enum: ['House', 'Apartment', 'None'],
+      },
+      numberOfSiblings: {
+        title: 'Number of Siblings',
+        type: 'string',
+        enum: ['0', '1', '2'],
+      },
+      submit: {
+        type: 'button',
+        title: 'Submit',
+      },
+    },
+  }
+  const uischema = {
+    type: 'VerticalLayout',
+    elements: [
+      {
+        type: 'Control',
+        scope: '#/properties/householdIncome',
+        options: {
+          classNames: classes.stackedControl,
+        },
+      },
+
+      {
+        type: 'Control',
+        scope: '#/properties/housing',
+        labelAlignment: 'top',
+      },
+      {
+        type: 'Control',
+        scope: '#/properties/numberOfSiblings',
+        labelAlignment: 'top',
+      },
+      {
+        type: 'Button',
+        scope: '#/properties/submit',
+      },
+    ],
+  }
+
+  const handleOnDone = ({ valid, value }) => {
+    console.log('handleOnDone clicked')
+    // value.parentId = `${point._id}`
+    // onDone({ valid, value })
+  }
+
+  const onSubmit = data => {
+    console.log('Form data:', data)
+    console.log('onSubmit clicked')
+    // Handle form submission logic here
+  }
+
+  return (
+    <div
+      className={cx(classes.container, className)}
+      {...otherProps}
+    >
+      <h3>We just need a few more details about you to get started.</h3>
+      <div className={classes.formContainer}>
+        <Column>
+          <JsonForms
+            schema={schema}
+            uischema={uischema}
+            data={data}
+            renderers={vanillaRenderers}
+            details={} // the data for jsonforms
```

Comment:
I had to say `details={{}}` to get it to compile.  In JSX details={} sets details to whatever is in the { }. To set it to an empty object then you have to put an empty object in their - details = { {} } 

After I made that change, I got this error:
```
schema is invalid: data/properties/submit/type must be equal to one of the allowed values, data/properties/submit/type must be array, data/properties/submit/type must match a schema in anyOf
Error: schema is invalid: data/properties/submit/type must be equal to one of the allowed values, data/properties/submit/type must be array, data/properties/submit/type must match a schema in anyOf
    at Ajv.validateSchema (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:6007:23)
    at Ajv._addSchema (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:6201:18)
    at Ajv.compile (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:5899:26)
    at coreReducer (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:705:27)
    at Object.defaultMiddleware [as current] (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:1485:62)
    at http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:8837:97
    at mountState (http://localhost:6006/vendors-node_modules_storybook_addon-a11y_dist_preview_mjs-node_modules_storybook_addon-essen-743d04.iframe.bundle.js:45786:20)
    at Object.useState (http://localhost:6006/vendors-node_modules_storybook_addon-a11y_dist_preview_mjs-node_modules_storybook_addon-essen-743d04.iframe.bundle.js:46331:16)
    at useState (http://localhost:6006/vendors-node_modules_storybook_addon-a11y_dist_preview_mjs-node_modules_storybook_addon-essen-743d04.iframe.bundle.js:58383:21)
    at JsonFormsStateProvider (http://localhost:6006/vendors-node_modules_jsonforms_vanilla-renderers_lib_jsonforms-react-vanilla_esm_js-node_modu-caca94.iframe.bundle.js:8837:76)
```
I don't think it likes the value 'button'.